CONTRIBUTING.md

```diff
@@ -74,7 +74,7 @@ Status labels attempt to capture the lifecycle of an issue:
 
 ### The documentation
 
-We use [MkDocs](https://www.mkdocs.org/) for [the language reference](https://crystal-lang.org/reference/).
+There's [a language reference](https://crystal-lang.org/reference/) available.
```

Comment:
Maybe something like this?

```markdown
The language reference is available at [https://crystal-lang.org/reference/](https://crystal-lang.org/reference/).
See the repository at [crystal-lang/crystal-book](https://github.com/crystal-lang/crystal-book) contributing to it.
```

Reply:
I think it would be the best
```markdown
[The language reference is available](https://crystal-lang.org/reference/).
See the repository at [crystal-lang/crystal-book](https://github.com/crystal-lang/crystal-book) for contributing to it.
```

Reply:
Or even `is available` part could be stripped of the link

Reply:
"The language reference is available" without a destination sounds weird. Sure it's available (what else would it be? 😄), but the relevant information is only in the context that the entire sentence is a link.

Reply:
Also the "a" or "the" should not be part of the link.